### ISSUE TITLE
Pip installation succeeds without installing dependencies

### ISSUE THREAD

**Issue by jkahn** [EducationalTestingService/rsmtool#26]
I think it should probably fail or at least specify that the dependencies need to be installed.
<details>

```
mkvirtualenv rsmtool-tester --python=/usr/bin/python3
cd $checkoutdir
pip install .
... # succeeds quietly
```

</details>

**Comment by jkahn** [comment]
... or make it work?

**Comment by desilinguist** [comment]
I'd rather not. I much prefer conda to pip now since it's much simpler and easier to use but I'm happy to accept a PR if anyone really wants it.

**Comment by jkahn** [comment]
`tox` virtualenvs still don't understand conda, and I still count on PyPI for distribution most of the time. 

I'm doing some tinkering with a requirements.txt file, and I wonder if you can clarify if any part of this is a library or if it's entirely an app? (Maybe a separate issue again.)

**Comment by desilinguist** [comment]
It's not that I haven't tried getting it to work with pip but I found that
doing so interfered with building conda packages which is unacceptable.

To answer your other question, RSMTool is also used as a library in some of
our ETS-internal applications.

**Comment by desilinguist** [comment]
This will probably work better when we upgrade to `statsmodels` to 0.8.1 which has a wheel available for all OSes and not just OS X.

**Comment by desilinguist** [comment]
Addressed by #178.
